src/main/java/net/minecraftforge/client/model/MultiModel.java

```diff
@@ -129,14 +129,14 @@ public Baked(ResourceLocation location, boolean perspective, @Nullable IBakedMod
             // Only changes the base model based on perspective, may recurse for parts in the future.
             if(base != null && perspective)
             {
-                ImmutableMap.Builder<TransformType, Pair<Baked, TRSRTransformation>> builder = ImmutableMap.builder();
+                EnumMap<TransformType, Pair<Baked, TRSRTransformation>> map = new EnumMap<>(TransformType.class);
                 for(TransformType type : TransformType.values())
                 {
                     Pair<? extends IBakedModel, Matrix4f> p = base.handlePerspective(type);
                     IBakedModel newBase = p.getLeft();
-                    builder.put(type, Pair.of(new Baked(location, false, newBase, parts), new TRSRTransformation(p.getRight())));
+                    map.put(type, Pair.of(new Baked(location, false, newBase, parts), new TRSRTransformation(p.getRight())));
                 }
-                transforms = builder.build();
+                transforms = ImmutableMap.copyOf(map);
```

Comment:
I'm not sure there is any advantage to the changes in this file.
Isn't using the `ImmutableMap.Builder` just fine?

Reply:
This way keeps the data in an `EnumMap` format.
It's only a minor difference, but as it's the same amount of code either way, I thought it might as well be done.

Reply:
Ok, I didn't know that `copyOf` had a special case for `EnumMap`. That's good to know.